.github/workflows/release.yml

```diff
@@ -13,26 +13,20 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-        
-      - name: Read version.properties
-        id: version
-        uses: juliangruber/read-file-action@v1
-        with:
-          path: ./version.properties
-        
-      - uses: rishabhgupta/split-by@v1
-        id: split
+
+      - uses: madhead/read-java-properties@latest
+        id: all
```

Comment:
`all` is meaningless here, because it's version.properties so this id could be `version`

Reply:
@luongvo Agree with you, it should be `version`.